.gitpod.yml

```diff
@@ -0,0 +1,8 @@
+image: gitpod/workspace-ruby-3.1
+tasks:
+  - init: |
+      # Code fetch for generating command guide
+      git clone --depth 1 -b 3.3 https://github.com/rubygems/rubygems.git ../rubygems || true &
+      bundle config set --local path vendor/bundle
+      BUNDLE_JOBS=16 bundle install
```

Comment:
`bundle install` by default uses as many processors available, so `BUNDLE_JOBS` seems unneeded too?

Reply:
Aha, I am still using both Bundler 2.2 and 2.3 so am still not sure when I can forget `BUNDLE_JOBS`. 😁 Fixed in the latest commit.